source/distributed/cell_data_transfer.inst.in

```diff
@@ -0,0 +1,36 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2018 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
+
+
+
+for (deal_II_dimension : DIMENSIONS; deal_II_space_dimension : SPACE_DIMENSIONS)
+  {
+#if deal_II_dimension <= deal_II_space_dimension
+    // we need this specialization to transfer active fe indices
+    template class parallel::distributed::CellDataTransfer<
+      deal_II_dimension,
+      deal_II_space_dimension,
+      std::vector<unsigned int>>;
```

Comment:
We introduced this specialization for `std::vector<unsigned int>>` to be able to transfer integers. We could also specialize for `dealii::Vector<unsigned int>`, but I think we will only use the integer transfer internally, so there is no need for it. What do you think?